infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/external/IngressServerExposer.java

```diff
@@ -102,6 +103,14 @@ private Ingress generateIngress(
       ingressBuilder = ingressBuilder.withHost(host);
     }
 
+    Map<String, String> annotations = new HashMap<>(preconfiguredAnnotations);
+    for (Map.Entry<String, String> entry : annotations.entrySet()) {
+      String value = entry.getValue();
+      if (value.contains("<<service-name>>")) {
```

Comment:
Can you please put the placeholder to a constant? Also wouldn't single `<>` be enough? `<service-name>` ? We use that in namespace placeholders.

Reply:
@sparkoo the suggestion is applied in https://github.com/eclipse-che/che-server/pull/56/commits/e6e5727a6e738ced8b8a19c85588d3de5785b8fb